extensions/amp-bodymovin-animation/0.1/amp-bodymovin-animation.js

```diff
@@ -73,16 +106,24 @@ export class AmpBodymovinAnimation extends AMP.BaseElement {
     return animData.then(data => {
       const opt_context = {
         loop: this.loop_,
+        autoplay: this.autoplay_,
         animationData: data,
       };
       const iframe = getIframe(
           this.win, this.element, 'bodymovinanimation', opt_context);
       return Services.vsyncFor(this.win).mutatePromise(() => {
         this.applyFillContent(iframe);
+        this.unlistenMessage_ = listen(
+            this.win,
+            'message',
+            this.handleBodymovinMessages_.bind(this)
+        );
         this.element.appendChild(iframe);
         this.iframe_ = iframe;
       }).then(() => {
-        return this.loadPromise(this.iframe_);
+        const loaded = this.loadPromise(this.iframe_);
+        this.playerReadyResolver_(loaded);
```

Comment:
we shouldn't call `playerReadyResolver_` here. actually we don't event need to listen to loadPromise anymore. we can just return `this. playerReadyPromise_` (since playerReadyPromise_ won't be resolved until iframe has loaded and lottie SDK has loaded as well)

Reply:
Done